frontend/src/App.tsx

```diff
@@ -59,9 +60,14 @@ const App = () => {
                     <Route path="/login" component={Login}></Route>
                     <PrivateRoute path="/accessdenied" component={AccessDenied}></PrivateRoute>
                     <PrivateRoute
-                      path="/admin"
```

Comment:
Replacing the administration dashboard for now with two routes that go directly to our pages.

Reply:
what's the reason for this?

Reply:
The dashboard doesn't need to exist at this point in time.  It also looks horrible and has zero functionality presently.  It will be cleaner to just go directly to the pages that have functionality.

Reply:
Sure, just know that I'm refactoring the routing and will likely bring a different routing later. It's fine for now